CHANGELOG.md

```
@@ -2,6 +2,115 @@
 
 All notable changes to cmux are documented here.
 
+## [0.64.17] - 2026-06-23
+
+### Added
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)